accepted/00000-hub-reporting.md

```diff
@@ -70,20 +56,50 @@ from the main Uyuni Server database.
 Data which belong to the traditional stack only (like osad status) should not be made available in the
 reporting database as the traditional stack will be deprecated soon.
 
+## Installation
+The reporting tools will be installed by default; all the upgrade to a newer version, will install the 
+reporting tools as well.
 
-## Adding Data to the Reporting DB
+We provide a new package installed in the hub system:
+- to setup the reporting DB schema,
+- to setup and manage the reporting database,
+- to setup and manage DBs in all Uyuni Server.
+
+The management tools should support:
+- Database setup and schema initialization
+- Database schema migration
+- Simple User management. The default setup will create two users:
+  * a Read/Write user, to manage the database and write the data from the main Uyuni Database into the Reporting DB. This user and the DB connection parameters are written into `/etc/rhn/rhn.conf` similar to the default DB options.
+  * a Read-Only user, used by the reporting tools for gathering information from the hub database.
+The package will also setup a taskomatic job to retrieve informations from all the other SUSE Manager Server
+database.
+All the other information required (e.g. Uyuni Server info) will be retrieved by already existing configuration.
+
+### Account management on the Hub for the Reporting Databases of the managed Servers
+
+During installtion, hub should create its own user on the Reporting Databases of the single Servers.
```

Comment:
Just to verify that I have understood this correctly, this is an additional users, created by the hub server on each servers in addition to the two default one mentioned in the paragraph above. And this is read only as well, I guess, since the hub does not need to change the data. I also would guess that the "Simple User management" tool mentioned above should not alter this hub account in any way, otherwise the sync will break.

Reply:
I tried to add more info but yes, probably we should rephrase the user part at all and make it more clear